packages/arb-token-bridge-ui/src/components/common/Layout.tsx

```diff
@@ -38,6 +41,8 @@ export function Layout(props: LayoutProps) {
         <main>{props.children}</main>
       </div>
 
+      <Toast />
```

Comment:
Can we comment out this+including the CSS, till the time it's not being used anywhere in the app?
Having Toast in the codebase is okay for me as it can be tree-shaken from the final build, but we should call it only when we're using it after replacing an alert. This will avoid potential errors from being shipped.